### DISCUSSION TITLE
Eachine e10

### DISCUSSION THREAD

**Discussion by hazcoper** [pascallanger/DIY-Multiprotocol-TX-Module#664] [Q&A]
Hi, does anyone know what protocol to use with this one?

https://productz.com/en/eachine-e10/p/lEwR

**Comment by pascallanger** [comment]
Try all sub protocols of CX10.

**Reply by hazcoper** [reply]
Tried all of them and i was not able to bind. Anything else worth trying?

**Reply by pascallanger** [reply]
All the protocols and sub protocols...

**Reply by hazcoper** [reply]
managed to find one that worked, its using aetr, and i had to flip the roll and yaw

the protocol that worked was the h8 3d std

**Reply by pascallanger** [reply]
Great, I've updated the documentation.
